packages/calling-component-bindings/src/devicePermissionSelector.ts

```diff
@@ -10,7 +10,7 @@ import { getDeviceManager } from './baseSelectors';
 
 export const devicePermissionSelector = reselect.createSelector([getDeviceManager], (deviceManager) => {
```

Comment:
Since there is no component designed for this selector, I've moved this to app folder, please also copy this change there

Reply:
Thanks, removed this file from calling-component-bindings.